### ISSUE TITLE
$hypergeometric_u vs %hypergeometric_u confusion

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3638]
Imported from SourceForge on 2024-07-08 10:30:59
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2021-03-06 12:33:17
Original: https://sourceforge.net/p/maxima/bugs/3734

---

Maxima knows nothing about hypergeometric_u , but specint sometimes uses that function--OK, fine. But there is a confusion between  %hypergeometric_u and  $hypergeometric_u

~~~
(%i32)	specint(%e^(-t)/k*sqrt(t*(t+2*k)),t);
(%o32)	2*sqrt(%pi)*hypergeometric_u(3/2,3,2*k)*k
(%i33)	% - 2*sqrt(%pi) * hypergeometric_u(3/2,3,2*k)*k;
(%o33)	2*sqrt(%pi)*hypergeometric_u(3/2,3,2*k)*k\-2*sqrt(%pi)*hypergeometric_u(3/2,3,2*k)*k
(%i34)	expand(%);
(%o34)	2*sqrt(%pi)*hypergeometric_u(3/2,3,2*k)*k\-2*sqrt(%pi)*hypergeometric_u(3/2,3,2*k)*k
(%i36)	ev(%,nouns);
(%o36)	0
~~~

I think specint should return  a $hypergeometric_u expression  not a % hypergeometric_u expression. Right?

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 10:31:01
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2021-03-16 05:21:14
Original: https://sourceforge.net/p/maxima/bugs/3734/#426b

---

- **labels**:  --> hypergeometric
- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 10:31:05
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2021-03-16 05:21:14
Original: https://sourceforge.net/p/maxima/bugs/3734/#6dde

---

Fixed by commit dc5a11f2.
